helm/README.md

```diff
@@ -0,0 +1,36 @@
+<!-- DO NOT EDIT. THIS IS GENERATED FROM README.md.gotmpl -->
+
+# Coder Helm Chart
+
+This directory contains the Helm chart used to deploy Coder onto a Kubernetes
+cluster. It contains the minimum required components to run Coder on Kubernetes,
+and notably (compared to Coder Classic) does not include a database server.
+
+## Getting Started
+
+> ⚠️ **Warning**: The main branch in this repository does not represent the
+> latest release of Coder. Please reference our installation docs for
+> instructions on a tagged release.
+
+View
+[our docs](https://coder.com/docs/coder-oss/latest/install#kubernetes-via-helm)
+for detailed installation instructions.
+
+## Values
+
+| Key                 | Type | Description | Default                         |
+| ------------------- | ---- | ----------- | ------------------------------- |
+| coder | object | Primary configuration for `coder server`. | `{"env":[{"name":"CODER_ACCESS_URL","value":"https://coder.example.com"}],"image":{"pullPolicy":"IfNotPresent","repo":"ghcr.io/coder/coder","tag":""},"resources":{},"service":{"enable":true,"externalTrafficPolicy":"Cluster","loadBalancerIP":"","type":"LoadBalancer"},"tls":{"secretName":""}}` |
```

Comment:
Do we need to do this generation? It always has confused me, and people can just look at values.yaml

Reply:
I'm fine with removing it and pointing towards values.yaml instead, @ammario any thoughts?

Reply:
I can see it both ways.

Reply:
I think we can remove it, but encourage users to install with defaults, such as the access URL and postgres URL